sections/semantics-forms.include

```diff
@@ -9053,29 +9470,56 @@ You cannot submit this form when the field is incorrect.</samp></pre>
-          <p>Group description: <strong>Layout/presentation on the WWW.</strong></p>
-          <p><meter value="0.5">Moderate activity,</meter> Usenet, 618 subscribers</p>
-        </li>
-        <li>
-          <p><a href="/group/netscape.public.mozilla.xpinstall/view">netscape.public.mozilla.xpinstall</a> - <a href="/group/netscape.public.mozilla.xpinstall/subscribe">join</a></p>
-          <p>Group description: <strong>Mozilla XPInstall discussion.</strong></p>
-          <p><meter value="0.25">Low activity,</meter> Usenet, 22 subscribers</p>
-        </li>
-        <li>
-          <p><a href="/group/mozilla.dev.general/view">mozilla.dev.general</a> - <a href="/group/mozilla.dev.general/subscribe">join</a></p>
-          <p><meter value="0.25">Low activity,</meter> Usenet, 66 subscribers</p>
-        </li>
-      </ul>
+      <section>
+        <div>
+          <h3>Suggested groups</h3>
+          <button onclick="hideSuggestedGroups()">Hide suggested groups</button>
+        </div>
+        <ul>
+          <li>
+            <p>
+              <a href="/group/comp.infosystems.www.authoring.stylesheets/view">comp.infosystems.www.authoring.stylesheets</a>
+              - <a href="/group/comp.infosystems.www.authoring.stylesheets/subscribe">join</a>
+            </p>
+            <p>
+              Group description:
+              <strong>Layout/presentation on the WWW.</strong>
+            </p>
+            <p>
+              <meter value="0.5">Moderate activity,</meter> Usenet, 618 subscribers
+            </p>
+          </li>
+          <li>
+            <p>
+              <a href="/group/netscape.public.mozilla.xpinstall/view">netscape.public.mozilla.xpinstall</a>
+              - <a href="/group/netscape.public.mozilla.xpinstall/subscribe">join</a>
+            </p>
+            <p>
+              Group description:
+              <strong>Mozilla XPInstall discussion.</strong>
+            </p>
+            <p>
+              <meter value="0.25">Low activity,</meter> Usenet, 22 subscribers
+            </p>
+          </li>
+          <li>
+            <p>
+              <a href="/group/mozilla.dev.general/view">mozilla.dev.general</a>
+              - <a href="/group/mozilla.dev.general/subscribe">join</a>
+            </p>
+            <p>
+              <meter value="0.25">Low activity,</meter> Usenet, 66 subscribers
+            </p>
+          </li>
+        </ul>
+      </section>
     </xmp>
-    Might be rendered as follows:
-    <img src="images/sample-meter.png" width="332" height="178" alt="With the <meter> elements rendered as inline green bars of varying lengths." />
+
+    with some light CSS, might be rendered as follows:
+
+    <p>
+      <img src="images/sample-meter.png" width="424" height="270" alt="With the meter elements rendered as inline green bars of varying lengths." />
+    </p>
```

Comment:
Please check that the image is licensed for republication.

Reply:
added note to the acknowledgements